workers/unity/Assets/Fps/Scripts/UI/Controllers/Testers/DeploymentListScreenTester.cs

```diff
@@ -0,0 +1,51 @@
+﻿using UnityEngine;
+
+namespace Fps
+{
+    [RequireComponent(typeof(DeploymentListScreenController))]
+    public class DeploymentListScreenTester : MonoBehaviour
+    {
+        public bool Test;
+        public int NumResultsToGenerate;
+
+        private void Update()
+        {
+            if (!Test)
+            {
+                return;
+            }
+
+            Test = false;
```

Comment:
Is the intention that during playmode you tick this box to trigger a refresh of the data?

Reply:
Yes